Rakefile

```diff
@@ -4,11 +4,14 @@ require 'bundler'
 require 'puppet_litmus/rake_tasks' if Bundler.rubygems.find_name('puppet_litmus').any?
 require 'puppetlabs_spec_helper/rake_tasks'
 require 'puppet-syntax/tasks/puppet-syntax'
-require 'puppet_blacksmith/rake_tasks' if Bundler.rubygems.find_name('puppet-blacksmith').any?
 require 'github_changelog_generator/task' if Bundler.rubygems.find_name('github_changelog_generator').any?
 require 'puppet-strings/tasks' if Bundler.rubygems.find_name('puppet-strings').any?
 require 'voxpupuli/acceptance/rake'
 
+# puppet-lint-top_scope_facts assumes all top scope variables are facts.
+# This configures the check to ignore all top scope variables listed in []
+PuppetLint.configuration.top_scope_variables = ['platform_tag']
```

Comment:
I don't think this changes anything, but to help follow the breadcrumbs, `platform_tag` is a fact defined in PE https://github.com/puppetlabs/puppet-enterprise-modules/blob/main/modules/puppet_enterprise/lib/facter/platform_tag.rb